### PR TITLE
backend: utilizing cached table for binance kline api data processing

### PR DESCRIPTION
**Pull request by thminhVN**
#### What's this PR do?

- [x] Technical solution for Utilizing cached table for Binance kline API data processing in Nghenhan

### CODE REVIEW

File: Engineering/Backend/Utilizing Cached Table for Binance Kline API Data Processing.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+tags: engineering/backend, nghenhan, hedge-foundation
```
Comment (zlatanpham) [inline Engineering/Backend/Utilizing Cached Table for Binance Kline API Data Processing.md:2]:
```suggestion
tags: engineering/backend, nghenhan, hedge-foundation, binance
```

**Review by zlatanpham — APPROVED** [review]
LGTM

File: Engineering/Backend/Utilizing Cached Table for Binance Kline API Data Processing.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+author: Minh Tran
+github_id: thminhVN
+date: 2023-05-07
+---
+
+# Introduction
```
Comment (zlatanpham) [inline Engineering/Backend/Utilizing Cached Table for Binance Kline API Data Processing.md:8]: Should down level all headings to 1 level

File: Engineering/Backend/Utilizing Cached Table for Binance Kline API Data Processing.md (hunk 1 above)
Comment (thminhVN) [reply Engineering/Backend/Utilizing Cached Table for Binance Kline API Data Processing.md]: added

File: Engineering/Backend/Utilizing Cached Table for Binance Kline API Data Processing.md (hunk 2 above)
Comment (thminhVN) [reply Engineering/Backend/Utilizing Cached Table for Binance Kline API Data Processing.md]: updated

**Review by monotykamary — APPROVED** [review]
LGTM

**Pull request merged**